### DISCUSSION TITLE
关于收费计划

### DISCUSSION THREAD

**Discussion by diyanqi** [oiso-developer-team/OIso-frontend#13] [General]
目前暂定本周六（3.4）正式开放 OIso Premium 注册，定价 5.9 元/月。然后下周六（3.11）推出安装油猴脚本+邀请三个用户赠送7天 Premium 计划（被邀请用户获得 3 天 Premium）。各位觉得这个营销计划如何（逃

具体时间可能会由于开发进度而调整，以上计划也请各位修改

**Comment by diyanqi** [comment]
@Lotuses-robot @george-jiang-wow

**Reply by george-jiang-wow** [reply]
定价￥5.78（逃
不失是一种营销策略。说实话如果爱发电没有5元底线我想收￥4.78
累计每签到20天送1~2天premium吧。别忘了签到页贴个或者弹出个支付宝红包

**Reply by george-jiang-wow** [reply]
另外，我个人建议在仔细审查无任何侵权、《用户协议》~~以及分赃~~的事情都商榷完成后再开始售卖。现阶段我认为应该先上线支付宝赚钱红包。 @diyanqi

**Reply by diyanqi** [reply]
> 另外，我个人建议在仔细审查无任何侵权、《用户协议》~以及分赃~的事情都商榷完成后再开始售卖。现阶段我认为应该先上线支付宝赚钱红包。 @diyanqi

要不您帮忙检查一下侵权和写一下用户协议？~然后再讨论分桩之事~

**Reply by george-jiang-wow** [reply]
> > 另外，我个人建议在仔细审查无任何侵权、《用户协议》~以及分赃~的事情都商榷完成后再开始售卖。现阶段我认为应该先上线支付宝赚钱红包。 @diyanqi
> 
> 要不您帮忙检查一下侵权和写一下用户协议？~然后再讨论分桩之事~

得考完之后了（

**Reply by george-jiang-wow** [reply]
> > 另外，我个人建议在仔细审查无任何侵权、《用户协议》~以及分赃~的事情都商榷完成后再开始售卖。现阶段我认为应该先上线支付宝赚钱红包。 @diyanqi
> 
> 要不您帮忙检查一下侵权和写一下用户协议？~然后再讨论分桩之事~

反正免费赞助和插广告都没大问题。就是收费得慢慢来，先搞用户体验、认可度。您要么去接个广告？
